smarts/core/argoverse_map.py

```diff
@@ -936,6 +936,21 @@ def _lanepoints(self):
         assert self._map_spec.lanepoint_spacing > 0
         return LanePoints.from_argoverse(self, spacing=self._map_spec.lanepoint_spacing)
 
+    def _resolve_in_junction(self, junction_lane: RoadMap.Lane) -> List[List[str]]:
+        # There are no paths we can trace back through the junction, so return
+        if len(junction_lane.road.incoming_roads) == 0:
+            return []
+
+        # Trace back to the road that leads into the junction
+        inc_road: RoadMap.Road = junction_lane.road.incoming_roads[0]
```

Comment:
Is this enough? I remember that some junction roads are split inside the intersection in SUMO. As in junction roads can lead into other junction roads so we would need to regress until the incoming road is not in the junction.

Reply:
I have not come across a case like like that so far. Argoverse lane segments tend to be quite long, and to go through the entire intersection. I think we should keep it simple for now.